misk/src/main/kotlin/misk/tracing/interceptors/TextMultimapExtractAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package misk.tracing.interceptors
+
+import io.opentracing.propagation.TextMap
+
+/**
+ * Provides an interface for Tracer.extract() to read from a multimap datasource
+ * (i.e. HTTP headers provided via OkHttp)
+ */
+class TextMultimapExtractAdapter(
+  private val multimap: Map<String, List<String>>
+) : TextMap {
+  override fun put(key: String?, value: String?) {
```

Comment:
nit: these are technically not nullable, are they?

Reply:
Good catch. Not sure what happened here